docs/components/Modal/modal.css

```diff
@@ -0,0 +1,39 @@
+
+.modal-container {
+    border: 1px solid black;
```

Comment:
suggestion: you should use color with variable name  to maintain the consistency with naming convention

Reply:
### sure bro